opinions/week6.md

```diff
@@ -0,0 +1,505 @@
+
+![https://cdn.discordapp.com/attachments/546525809440194560/1076855413246533693/image.png](https://cdn.discordapp.com/attachments/546525809440194560/1076855413246533693/image.png)
+
+This is the popular technique in Atonal Music Composition, namely, 12-tone row. The idea is to distribute the notes such that there is no note is emphasizing the individual key, which is to use all of the 12 tones possible. This also gives the interesting sounds which can represent the shine (and the “essence”) of the stars in the sky.
+
+### Genre Changes every 8 bars (takehirotei)
+
+![Screenshot_365.png](https://cdn.discordapp.com/attachments/546525809440194560/1079355206086709339/Screenshot_365.png)
+
+For the genre decision,  we basically just choose the genres that are suitable in high bpm, so in this case, we choose hardcore, chiptune, xi (yes, xi is a genre), artcore, metal, frenchcore and speedcore. As for deciding their placement, we need to find genres that can be transition smoothly to each other. Hence, hardcore was used as the first genre since it can literally transition into anything, then follow by chiptune and xi. xi and artcore have some similarity, hence we can put artcore after xi. As metal can again, transition into anything due to its wide use, we put it between artcore and hardcore. After that, frenchcore, speedcore and hardcore is pretty similar, so we just put them back to back.
+
+### BPM Changes every 4 bar* (HowToPlayLN)
+
+The BPM changes in this part are applied every 4 bars, and they are all, **Prime Numbers** from 223 to 317.
+
+### Totally Random Notes (HowToPlayLN)
+
+In the transition from 251bpm to 257bpm, there are the triplets played, which is this:
+
+![https://cdn.discordapp.com/attachments/546525809440194560/1076857209117474887/image.png](https://cdn.discordapp.com/attachments/546525809440194560/1076857209117474887/image.png)
+
+This is totally random, and there is no further explanation (maybe similar to 12-tone and the chord actually addresses it? idk)
+
+### onostrapha reference (HowToPlayLN)
+
+The same part, before transitioning to metal, this is the melody played
+
+![https://cdn.discordapp.com/attachments/546525809440194560/1076857957083512852/image.png](https://cdn.discordapp.com/attachments/546525809440194560/1076857957083512852/image.png)
+
+This is a homage to **rN - onostrapha,** the osu! world cup 2022 custom song. rN used the quintuplet, following with septuplet in the piano intro before the trills. This also gives the “twist” and preparation for the next part.
+
+## Forever
+
+### The Lost Dedication to Cryochemistry (HowToPlayLN)
+
+**HowToPlayLN** In the last piano arrangement, I cited **The Lost Dedicated** and **Cryochemistry** into the arrangement. Especially Cryochemistry, his song was really my inspiration into making a tiebreaker music, as well as inspired me a lot in storyboarding, and providing me the huge motivation to push this community forward.
```

Comment:
probably you mean "this" after "Especially cryochemistry"?

Reply:
oh yea, that was a typo, will edit in the next revision